src/main/routes/scenarios.js

```diff
@@ -1,6 +1,16 @@
 const express = require('express');
 const router = express.Router();
 
+//Route firebaseAdmin
+const admin = require('firebase-admin');
```

Comment:
Is it possible to create a file for this const and export it?  

Something like: src/db/bd.js

Because we need to use this const in several files

Reply:
I think it's posible, i will check and later make that change.